src/pieces/commands/commands_functions.py

```diff
@@ -124,8 +124,6 @@ def list_assets(list_type_or_max='assets', **kwargs):
         return
 
     if list_apps:
-        # Logic for listing applications
-        double_space("Listing applications...")
```

Comment:
any reason why we removed this?

Reply:
Consistently
Why all assets models listed without saying "listing ...."

Reply:
assuming b/c comparing the output in the unit test

Reply:
as long as this is consistent we are gucci